HW11.py

```diff
@@ -0,0 +1,102 @@
+"""
+Homework 11. Bank Card operations.
+
+# write class for bank card.
+# Class should reflect card lifecycle, card operations etc.
+# use CVV, PIN, Name, Surname , end date, card_num as initial params
+# class should have in addition to common logic some class attributes,
+# as minimum one classmethod and as minimum  one staticmethod,
+# two or more getters/setters, __str_ magic method
+# do not forget about block if __name__ == '__main__':
+# and code there to check your class written logic.
+"""
+
+from datetime import datetime as dt
+import re
+
+
+class CardOperations:
+    CVV_DIGITS = 3
+    card_type = "Standard"
+
+
+    def __init__(self, CVV, Card_num, EndDate, Surname, Name, PIN, Money):
+        self.CVV = CVV if self.check_CVV(CVV) else "000"
+        self.PIN = PIN
```

Comment:
cvv, pin and card_num are sensitive information members - we need to make them private and organize getter/setters